### PR TITLE
[I18N] Updated Lato font to support Cyrillic, Greek-​​based languages

### PR DESCRIPTION
**Pull request by dvska**
### Description of the issue/feature this PR addresses:
Improper look of the, at least, POS interface

### Current behavior before PR:
Lack of the cyrillic, greek symbols

### Desired behavior after PR is merged:
Proper look of web-pages using Lato font;
Additionally, faster load due to adding .woff2 format

` `
Font files origin is http://www.latofonts.com/download/Lato2OFLWeb.zip

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by mart-e** [comment]
Hello,

I am not sure about this one. Should we load a new font for every user, even the one not using Cyrillic languages?

cc @ged-odoo

**Comment by ged-odoo** [comment]
@mart-e i did not really check the PR, but on principle, a common font which supports more languages is ok.  We can afford to have a larger font footprint

**Comment by mart-e** [comment]
To be clear, the font folder goes from 2.5 to 34M, not a small change (size increase partially explained by the addition of the woff2 files).
But maybe the browser only fetches when needed and it is not important.

@dvska Could you add before/after screenshot to see the difference?

**Comment by dvska** [comment]
@mart-e
odoo-pr-latto-font

odoo-pr-latto-font

**Comment by mart-e** [comment]
Ok so 34MB for the font folder is probably a bit too much.
According to https://transfonter.org/formats, can't we drop some formats?

If we just keep .woff and .woff2 formats (supported by all browsers that Odoo supports, we reach a font folder or 4.8MB (from 396K without the other heavy formats).

@antonylesuisse you should probably decide here.
**TL;DR**: add support to way more alphabets in Lato font, at the cost of making the fonts files bigger.

**Comment by dvska** [comment]
@ged-odoo @mart-e @antonylesuisse 
Agreed with *mart-e*, cleaned out legacy .eot, .ttf, .svg files, now the `Lato` fonts folder is just 4.7Mb.

P.S. Should I prepare a separate PR to remove such legacy font files from entire Odoo codebase?

**Comment by mart-e** [comment]
@dvska I know this is a pretty old issue but I believe I have a solution at #32312
Would you mind test it?
You can use a runbot on http://runbot.odoo.com/runbot/repo/git-github-com-odoo-odoo-1?search=32312
